plugins/linear-genome-view/src/LinearGenomeView/components/Highlight.tsx

```diff
@@ -25,19 +25,18 @@ const useStyles = makeStyles()(theme => ({
   highlight: {
     height: '100%',
     position: 'absolute',
-    background: `${colord(theme.palette.quaternary?.main ?? 'goldenrod')
+    overflow: 'hidden',
```

Comment:
any reason for overflow: hidden being added here?

Reply:
without it, the icon on the highlight always shows (so if you're zoomed out and the highlight is a sliver you see it)

realistically, anything you're highlighting and looking at, you'd be a close enough zoom level to make use of the icon options, i think

Reply:
sounds good. i had this idea that possibly the link icon could even be hidden if the user is not hovering over, as the user might like it hidden for e.g. screenshot. just an idea though